apps/admin_app/lib/admin_app_web/exporter/exporter.ex

```diff
@@ -0,0 +1,93 @@
+defmodule AdminAppWeb.Exporter do
+  @moduledoc """
+  Data export module for Snitch.
+  """
+
+  alias AdminAppWeb.Exporter.Product, as: ProductEx
+  alias AdminAppWeb.Exporter.Order, as: OrderEx
+  alias Snitch.Domain.Order, as: Domain
+  alias Snitch.Core.Tools.MultiTenancy.Repo
+  alias Elixlsx.{Workbook, Sheet}
+  alias Snitch.Data.Schema.{Order, Product}
+  alias AdminAppWeb.DataExportMail
+
+  defp get_columns(type) do
+    case type do
+      "order" ->
+        ~w(id number line_items_count order_total billing_address shipping_address inserted_at updated_at user_id state)a
```

Comment:
@jyotigautam I see this as repetition from [line](https://github.com/aviacommerce/avia/pull/376/files#diff-1fe76619644f2dc3a5d5e02bdc3ebb5eR11).

Same goes for line 20 for production options. 

This violates the [DRY](https://metova.com/dry-programming-practices/) principle. 

Am I missing something here? Please propose a solution for this if my understanding is correct.

Reply:
I forgot to remove this. I have passed column list for product and order as arguments from product exporter and order exporter function respectively.